canonical-websites/snapcraft.io
# Updated active devices colour scale

# Done

Fixes https://github.com/canonical-websites/snapcraft.io/issues/290

- Added qualitative scale from http://colorbrewer2.org/#type=qualitative&scheme=Paired&n=11

# QA

- Pull this branch
- `./run`
- Visit a snap with data: https://0.0.0.0:8004/account/snaps/SNAP_NAME/measure
- See the new graphs

# Screenshots

![screenshot from 2018-03-07 09-51-22](https://user-images.githubusercontent.com/479384/37085548-3fc1344a-21ed-11e8-8eeb-09a5b7409a07.png)
